modules/users/client/config/users.client.config.js

```diff
@@ -28,10 +28,3 @@ angular.module('users').config(['$httpProvider',
     ]);
   }
 ]);
-
-angular.module('users').constant('settings', {
```

Comment:
Pretty sure these are no longer used now that I've deleted the frontend angular directive (which was never used).